### PR TITLE
Update hero pages with new benchmarks

### PR DESCRIPTION
**Pull request by Argn0**

### CODE REVIEW

File: src/components/Hero/BenchmarkGraphs.jsx [hunk 1]
```diff
@@ -17,7 +17,7 @@ const Wrapper = styled.div`
   }
 `;
 
-const mapData = (data, key) => data.map(item => ({ Percentage: (`${item.percentile * 100}%`), Value: Number(item[key].toFixed(2)) }));
+const mapData = (data, key) => data.map(item => ({ Percentage: (`${item.percentile * 100}%`), Value: typeof item[key] === 'number' && Number(item[key].toFixed(2)) }));
```
Comment (howardchung) [inline src/components/Hero/BenchmarkGraphs.jsx]: what's the additional check for?  Are any of them not numbers?

File: src/components/Match/matchColumns.jsx [hunk 2]
```diff
@@ -332,7 +332,7 @@ export default (strings) => {
         if (match.version || !parsedBenchmarkCols.includes(key)) {
           cols.push({
             displayName: strings[`th_${key}`] || strings[`heading_${key}`],
-            tooltip: strings[`tooltip_${key}`],
+            tooltip: strings[`benchmarks_${key}`] || strings[`tooltip_${key}`],
```
Comment (howardchung) [inline src/components/Match/matchColumns.jsx]: I think we should remove the benchmarks_ strings and replace them with tooltip_ strings (many already exist)

File: src/components/Match/matchColumns.jsx [hunk 3]
```diff
@@ -332,7 +332,7 @@ export default (strings) => {
         if (match.version || !parsedBenchmarkCols.includes(key)) {
```
Comment (howardchung) [inline src/components/Match/matchColumns.jsx]: do you need to update parsedBenchmarkCols for stuns_per_min?

File: src/components/Hero/BenchmarkGraphs.jsx (hunk 1 above)
Comment (Argn0) [reply src/components/Hero/BenchmarkGraphs.jsx]: Without this the page will keep crashing for a while after we add new benchmarks

**Comment by Argn0** [comment]
-removed all the benchmarks strings that were added recently
-updated parsedBenchmarkCols array
-added new string for stuns_per_min

**Comment by howardchung** [comment]
https://stage.opendota.com/heroes/102/benchmarks looks like the table header string is missing

**Pull request merged**